genesyscloud/architect_schedulegroups/data_source_genesyscloud_architect_schedulegroups_test.go

```diff
@@ -2,7 +2,7 @@ package architect_schedulegroups
 
 import (
 	"fmt"
-	"terraform-provider-genesyscloud/genesyscloud"
+	archiSchedules "terraform-provider-genesyscloud/genesyscloud/architect_schedules"
```

Comment:
Small thing.  I would change this to architect_schedules instead of archischedules

Reply:
`architectSchedules` would be better, we try to avoid snake case

Reply:
Ok, I will update the code with "architectSchedules".